package.json

```diff
@@ -21,7 +21,7 @@
     "setup": "npm install && npm run flow-typed",
     "test": "npm run lint && npm run flow-typed && npm run flow && npm run jest && npm run karma",
     "webpack": "babel-node --plugins=transform-es2015-modules-commonjs ./node_modules/.bin/webpack --progress",
-    "jest": "jest test/server --env=node --no-cache --collectCoverageFrom='server/' --coverage --verbose --runInBand --silent=false",
+    "jest": "NODE_ENV=local jest test/server --env=node --no-cache --collectCoverageFrom='server/' --coverage --verbose --runInBand --silent=false",
```

Comment:
I think jest sets NODE_ENV to "test" by default. Can we stick with that and mock out this value for the specific test that is expecting `NODE_ENV === "test"` to be true?

Reply:
@gregjopa That's correct.  It is set to test.  Not sure how this matters, but will see about overriding for that test.